http/timer_queue.hpp

```diff
@@ -15,7 +15,7 @@ constexpr const size_t timerQueueTimeoutSeconds = 5;
 namespace detail
 {
 
-constexpr const size_t maxSize = 100;
+constexpr const size_t maxSize = 300;
```

Comment:
Can you push this upstream?

Reply:
I had started a discord discussion on this. Awaiting responses from Ed.

Reply:
How did we decide 300 vs like 200?

Reply:
I am concerned about other limits like memory of having 300 requests in flight

Reply:
300 was a random number which i came up with. I dont have the expertise there to decide what that number should be.
@gtmills Can you please explain the need of this timerQueue and how that works on bmcweb ?

Reply:
I am going to keep going on this but approved this PR